### PR TITLE
[Go] Add Gearbox

### PR DESCRIPTION
**Pull request by muhammednagy**
Gearbox is a web framework written in Go with a focus on high performance and memory optimization.

### CODE REVIEW

File: go/gearbox/config.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+framework:
+  website: gogearbox.com
+  version: 1.0.0
```
Comment (waghanza) [inline go/gearbox/config.yaml:3]: could you use `X.Y` format ? e.g -> `1.0`

File: go/gearbox/config.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+framework:
+  website: gogearbox.com
+  version: 1.0.1
```
Comment (waghanza) [inline go/gearbox/config.yaml:3]: Could you use `1.0` ?

**Comment by muhammednagy** [comment]
Done @waghanza

**Comment by muhammednagy** [comment]
Sure. @waghanza 
That'd be my honour :)

**Pull request merged**